tests/dopamine/discrete_domains/run_async_training_test.py

```diff
@@ -111,7 +111,15 @@ def _put_call_cnt(v):
     self.assertEqual(_put_call_cnt(None), 2)  # Stop task.
 
   def testNumberSteps(self):
```

Comment:
Suggsetion here to separate each of these checks into its own test.

Reply:
done.